tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -513,6 +513,26 @@ TEST_P(RDFSimpleTests, BookCustomAction)
    EXPECT_EQ(*maxSlot, nWorkers-1);
 }
 
+TEST(RDFSimpleTests, StandardDeviation)
+{
+   RDataFrame rd1(8);
+   int i(0);
+   auto stdDev = rd1.Define("b1",
+                            [&i]() {
+                               ++i;
```

Comment:
this is not thread-safe

Reply:
Fixed in 3e06584fa051988ffa8e28ddd6619f9c2ba885ce